src/Microsoft.ML.StandardLearners/Standard/Simple/SimpleTrainers.cs

```diff
@@ -211,12 +236,16 @@ public sealed class Arguments
         public override TrainerInfo Info => _info;
 
         public PriorTrainer(IHostEnvironment env, Arguments args)
```

Comment:
>public [](start = 8, length = 6)

private

Reply:
Why should I make the constructor private?

---
In reply to: [218154524](https://github.com/dotnet/machinelearning/pull/876#discussion_r218154524) [](ancestors = 218154524)